test/e2e/dedicated_admin_test.go

```diff
@@ -0,0 +1,89 @@
+//+build e2e
+
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	rbacv1 "k8s.io/api/rbac/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var _ = Describe("Openshift on Azure dedicated-cluster-admin e2e tests [DedicatedAdmin]", func() {
+	defer GinkgoRecover()
+
+	It("should read nodes", func() {
+		_, err := dadmin.kc.CoreV1().Nodes().Get("master-000000", metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("should have full access on all non-infrastructure namespaces", func() {
+		// Create project as normal user
+		namespace := nameGen.generate("e2e-test-")
+		c.createProject(namespace)
+
+		wait.PollImmediate(2*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
Return the error from here and check that it's nil

Reply:
added